src/objects/organism_object.py

```diff
@@ -24,7 +22,49 @@ class OrganismObject:
     connections between the elements.
     """
     
-    def __init__(self, _id: str, conf: dict) -> None:
+    def __init__(self, _id: str, config: dict = None, mode: str = "SCANNER") -> None:
```

Comment:
I thought we talked about defining a mode as default and that the most appropriate one was SCANNER, which is the one that needs the least input information and in the case of using the constructor incorrectly the user would notice because it does not contain all the information to be able to perform a motif discovery execution.

But you are right that it is more appropriate for the user to indicate the execution mode in any case.

Translated with DeepL.com (free version)